dev/ci/user-overlays/08671-mattam-plugin-tutorials.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "$CI_PULL_REQUEST" = "8671" ] || [ "$CI_BRANCH" = "evar_info-flags" ]; then
+    plugin_tutorial_CI_REF=pr8671-fix
+    plugin_tutorial_CI_GITURL=https://github.com/mattam82/plugin_tutorials
+
+fi
```

Comment:
The PR number and the branch name in this overlay are not the ones of the present PR, thus they won't be selected.

Reply:
Right, thanks